src/main/resources/logback-spring.xml

```diff
@@ -46,6 +47,18 @@
         <appender-ref ref="consoleAppender"/>
     </logger>
 
+    <logger name="org.flywaydb" additivity="false" level="WARN">
+        <appender-ref ref="consoleAppender"/>
+    </logger>
+
+    <logger name="org.hibernate" additivity="false" level="WARN">
+        <appender-ref ref="consoleAppender"/>
+    </logger>
+
+    <logger name="com.zaxxer" additivity="false" level="WARN">
```

Comment:
I need to look into `com.zaxxer` to find out what it actually is related too

Reply:
I believe this is for the hikari connection pool

Reply:
It is indeed